### DISCUSSION TITLE
403 forbidden

### DISCUSSION THREAD

**Discussion by Dan9704** [iterate-ch/cyberduck#15941] [Q&A]
HI i got into some trouble
- my .htaccess file just auto being deleted
- I cant upload or get access to anything in my cyberduck accounts
can you please help me out

<img width="530" alt="image" src="https://github.com/iterate-ch/cyberduck/assets/116186495/dc895f76-08a9-4b3c-8f2b-4fd2a7a824f4">
<img width="929" alt="image" src="https://github.com/iterate-ch/cyberduck/assets/116186495/249fb00d-50ce-4b8f-bee2-4b03515d51fd">

**Comment by tatze96** [comment]
Please contact your server administrator to check if there is a server-side issue that is causing the file access and transfer issues.

If there aren't server-side issues [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issues, and send us the log file afterward for further investigation. If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15941 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
